joint_trajectory_controller/test/joint_trajectory_controller_test.cpp

```diff
@@ -981,6 +981,9 @@ TEST_F(JointTrajectoryControllerTest, ignorePartiallyOldActionTraj)
 }
 
 // Velocity FF parameter ///////////////////////////////////////////////////////////////////////////////////////////////
+// This test will only be built and run for the VelocityJointInterface-based version of the JointTrajectoryController
+
+#ifdef TEST_VELOCITY_FF
```

Comment:
Please consider `#if TEST_VELOCITY_FF` to handle `TEST_VELOCITY_FF=0` as well.

Reply:
Just noticed this comment. Updated.